Ical.Net/VTimeZoneInfo.cs

```diff
@@ -90,15 +89,15 @@ public virtual string TzId
     ///     </list>
     /// </example>
     /// </summary>
-    public virtual string TimeZoneName
+    public virtual string? TimeZoneName
     {
         get => TimeZoneNames.Count > 0
             ? TimeZoneNames[0]
             : null;
         set
         {
             TimeZoneNames.Clear();
-            TimeZoneNames.Add(value);
+            TimeZoneNames.Add(value ?? string.Empty);
```

Comment:
Looking at the getter, `null` would represent an empty list rather than a list with an empty string as its sole member.